src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -16,8 +16,9 @@ public class SendPacketsAsync
         private readonly ITestOutputHelper _log;
 
         private IPAddress _serverAddress = IPAddress.IPv6Loopback;
-        // In the current directory
-        private const string TestFileName = "NCLTest.Socket.SendPacketsAsync.testpayload";
+        // Accessible directories for UWP app:
+        // C:\Users\<UserName>\AppData\Local\Packages\<ApplicationPackageName>\
+        private string TestFileName = Environment.GetEnvironmentVariable("LocalAppData") + @"\NCLTest.Socket.SendPacketsAsync.testpayload";
```

Comment:
I tried on my machine, seems "corefx\bin\AnyOS.AnyCPU.Debug\System.Net.Sockets.Tests\netstandard" doesn't contain "LocalApplicationData" folder, and returns System.IO.DirectoryNotFoundException